docker_interop/docker-compose.yml

```diff
@@ -0,0 +1,122 @@
+version: '3'
+services:
+  besu-node-0:
```

Comment:
Could we change this to be 2 besu and 2 quorum nodes. So that we can see that Besu nodes can talk to each other as well and make it a be more balanced.